docker/agent/Dockerfile.agent

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE='adoptopenjdk/openjdk8:alpine'
+ARG BASE_IMAGE='adoptopenjdk/openjdk11:alpine'
```

Comment:
Do you upgrade the agent's JDK? I am good for moving OAP to JDK11.

Reply:
> Do you upgrade the agent's JDK? I am good for moving OAP to JDK11.

This is only a default value and can be overrode, it's good to me, we will finally release multiple Docker images based on different JDK version

Reply:
agent image's JDK is not used, right?  except users use agent image as their java application runtime base image. @kezhenxu94

Reply:
> agent image's JDK is not used, right? except users use agent image as their java application runtime base image. @kezhenxu94

Yes right

Reply:
I am curious, where do we use this agent image? If not, why upgrade? Do we plan to use it somewhere?
JDK8 is the version we announced the minimal supported JVM version, and still in maintain mode by OpenJDK and vendors. Why are we going rid of it now?

Reply:
> JDK8 is the version we announced the minimal supported JVM version

If we have this consideration, then we don't need to update it, I will revert this change.

Reply:
Thanks.